NOTICE

```diff
@@ -29,6 +29,8 @@ under the licensing terms detailed in LICENSE:
 * Valeria Viana Gusmao <valeria.viana.gusmao@gmail.com>
 * Gabor Greif <ggreif@gmail.com>
 * Martin Fredriksson <martin.fredriksson@vikinganalytics.se>
+* Joe Pea (@trusktr) <trusktr@gmail.com>
+* Joe Pea (@trusktr) <joe@trusktr.io>
```

Comment:
It would be good to pick one and stick with the format because the NOTICE file is also parsed to generate contributors in package.json for example.